src/libs/Navigation/Navigation.js

```diff
@@ -158,6 +159,11 @@ function navigate(route = ROUTES.HOME) {
         return;
     }
 
+    // Pressing a button that does navigation will stay focused, keeping its tooltip visible even if it's supposed to be out of view.
+    // To prevent that we blur the button manually (especially for Safari, where the mouse leave event is also missing).
```

Comment:
A pressed navigation button will remain focused, keeping its tooltip visible, even if it's supposed to be out of view. To prevent that we blur the button manually (especially for Safari, where the mouse leave event is missing).

Reply:
Would you mind just modifying the comment a little bit please

Reply:
Updated